content/vendor/additional-objects/config.md

```diff
@@ -0,0 +1,8 @@
+---
+date: 2019-11-27
+linktitle: "Config"
+title: Config
+weight: 20020
+---
+
+Config object specifications control the presentation of a [custom configuration page](/vendor/config/config-screen/), enabling operators to specify any required or optional application-specific config settings at install time, or update those options later through the kotsadm dashboard.
```

Comment:
enabling operators — is the operator here the application developer? If so, we should define who this role is.

Reply:
"Config object specifications control the presentation of a [custom configuration page](/vendor/config/config-screen/), enabling application developers to specify any required or optional application-specific config settings for the cluster operator to provide at install time, or update those options later through the Admin Console."